crates/storage/provider/src/post_state.rs

```diff
@@ -848,8 +694,8 @@ mod tests {
 
         // Delete account A
         let mut post_state = PostState::new();
-        post_state.destroy_account(address_a, Account::default());
-        post_state.write_to_db(&tx, 1).expect("Could not write post state to DB");
+        post_state.destroy_account(2, address_a, Account::default());
+        post_state.write_to_db(&tx).expect("Could not write post state to DB");
 
```

Comment:
before the transitions would be 0-indexed, but now we key by block number which starts from 1 (in the test)?

Reply:
Yes, that is correct, as we now directly use BlockNumber to index change, and our block starts at `1`